netket/random.py

```diff
@@ -46,20 +46,20 @@ def uniform(low=0.0, high=1.0):
     return _np.random.uniform(low, high)
 
 
-@jit
-def randint(low, high):
+def randint(low, high, size=None):
```

Comment:
What was the rationale for `netket.random`? Do wee really need it? it's basically calling numpy

Reply:
@gcarleo I still don't see why it's needed. Can't we directly call into numpy from the samplers, or there is a weird limitation from numba?